config.json

```diff
@@ -152,7 +152,13 @@
     "peerStore": {
       "path": "./p2pstore"
     },
-    "reconnectInterval": "30s"
+    "reconnectInterval": "30s",
```

Comment:
Don't forget to add these parameters to all other available config files.
Maybe we also can remove `config_as.json` ?

Reply:
Can be done in another commit